### PR TITLE
Initial structure for Jupyter notebooks and Python helper code

### PR DESCRIPTION
**Pull request by ronaldfw**
We want to add Jupyter notebook examples for prototyping and
deeper analysis of capture data. We add an initial structure here that
outlines most of the parts needed, including Python helper code
(in orbitutils), unit tests for Python helper code, a build script
(in Python so it works on Windows and Linux) that compiles the needed
protobuffers and runs all unit tests for verification, and a sample
notebook so users can verify their setup.

### CODE REVIEW

File: contrib/jupyter/build.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# We need to specify the "root" of the src tree. With this choice, we can then
+# import the proto as "import ClientProtos.capture_data_pb2.py"
+proto_path = os.path.join(file_directory, "../../src/")
+
+# This is the only .proto file we currently need for our python code.
+capture_data_proto = os.path.join(file_directory,
+                                  "../../src/ClientProtos/capture_data.proto")
+
+command = "protoc --python_out=" + orbitutils_path + " --proto_path=" + proto_path + " " + capture_data_proto
+os.system(command)
```
Comment (beckerhe) [inline contrib/jupyter/build.py:22]: What about stopping here if `protoc` fails?

**Review by beckerhe — APPROVED** [review]
Otherwise LGTM.

One more thing: Is it intended that the capture file is empty?

File: contrib/jupyter/README.me [hunk 2]
```diff
@@ -0,0 +1 @@
+Instructions to come
```
[the remark this answers is not in the document]
Comment (ronaldfw) [reply contrib/jupyter/README.me]: Ha, yes, of course. Fixed :)

File: contrib/jupyter/build.py (hunk 1 above)
Comment (ronaldfw) [reply contrib/jupyter/build.py]: Yes, good point. Fixed.

**Comment by ronaldfw** [comment]
> Otherwise LGTM.
> 
> One more thing: Is it intended that the capture file is empty?

Hmm, no, that's not intended. Let me check.

**Comment by ronaldfw** [comment]
> > Otherwise LGTM.
> > One more thing: Is it intended that the capture file is empty?
> 
> Hmm, no, that's not intended. Let me check.

In the diff the file is shown as empty, but when I click on "View file" it actually shows the correct size (1.13MB):
https://github.com/google/orbit/blob/5ac88e1fa955aef7a6bcd4a0dd551f96d02b48ae/contrib/jupyter/orbitutils/testdata/OrbitTest_capture.orbit

**Comment by ronaldfw** [comment]
Thanks for the review, @beckerhe !

**Pull request merged**